tsconfig.base.json

```diff
@@ -12,13 +12,17 @@
     "typeRoots": ["node_modules/@types", "./typings"],
     "baseUrl": ".",
     "paths": {
-      "@fluentui/set-version": ["packages/set-version/src/index.ts"],
       "@fluentui/react-conformance": ["packages/react-conformance/src/index.ts"],
       "@fluentui/react-utilities": ["packages/react-utilities/src/index.ts"],
```

Comment:
```suggestion
```

Can we also delete a this one? If there is a dependency on it somewhere, please let me know and I will remove it in meantime 🔪